websites/M/MatchBox/metadata.json

```diff
@@ -0,0 +1,31 @@
+{
+	"$schema": "https://schemas.premid.app/metadata/1.10",
+	"author": {
+		"id": "596877893137596416",
+		"name": "c_fornow"
+	},
+	"service": "MatchBox",
+	"description": {
+		"en": "MatchBox is a skill based matchmaking service for Super Smash Bros. Ultimate"
+	},
+	"url": [
+		"thematchbox.gg",
```

Comment:
Is this an official domain for this service? It currently just shows a landing page

Reply:
https://thematchbox.gg/ works for me, I get a landing page at "www.thematchbox.gg" which is why its not included in the urls

Reply:
Oops, my bad. I didn't realize that I put www in the url bar when testing it...